LICENSE.md

```diff
@@ -0,0 +1,21 @@
+MIT License
+
```

Comment:
Is this the same as the C-SDK repository?

Reply:
Yes, it's identical